src/components/Profile/components/VictoryPromise/components/VictoryPromiseIcons.tsx

```diff
@@ -44,10 +43,8 @@ const VictoryPromiseIcon = ({
   );
 };
 
-type PrimaryColor = VictoryPromiseColor | typeof gordonColors.primary.blue;
-type SecondaryColor =
-  | typeof gordonColors.neutral.gray
-  | (typeof gordonColors.neutral.grayShades)[50];
+type PrimaryColor = VictoryPromiseColor | '#014983';
```

Comment:
I'm not convinced this is the same, since it was doing a typeof call before.  Could you tell what this was doing previously?